docs/experiments/experiment1.md

```diff
@@ -1,9 +1,104 @@
 ---
 
 id: experiment1
-title: First Experiment
-sidebar_label: Data Sets
+title: Standard Search vs. Stemming
+sidebar_label: Stemming
 
 ---
 
-Experiments are coming!
\ No newline at end of file
+For our first experiment we connected a Google Colab Notebook to our Elasticsearch instance and started comparing different build-in search metrics of Elasticsearch.
+
+Therefore, we parsed [these 5 free available corpora](http://ir.dcs.gla.ac.uk/resources/test_collections/) hosted by the University of Glasgow and feed them to the Elasticsearch [Ranking API](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-rank-eval.html). With [our parsing guide](https://pragmalingu.de/docs/guides/how-to-parse) you can try this yourself or you can look at our [Data Set Notebooks.]().
+
+We tried comparing the following operators:
+* [Match query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html) 
+* [Simple query string](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html)
+* [Standard english stemmer provided by Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-stemmer-tokenfilter.html)
+
+
+## Match operator
+
+The [match query operator](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html) is the standard search operator in Elasticsearch. A full-text search in the given fields of the documents comes with standard tokenization and an analyzer.
+
+The following is an example of a request that searches with match query. It operates on the query string in the 'text' field of the medline corpus:
+
+```
```

Comment:
if you add ```json 
than the code will be correctly highlighted

Reply:
I added python since the code from the notebook is in in python